examples/digital_fingerprinting/production/grafana/run.py

```diff
@@ -0,0 +1,533 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""DFP training & inference pipelines for Azure Active Directory logs."""
+
+import functools
+import logging
+import logging.handlers
+import multiprocessing
+import os
+import typing
+from datetime import datetime
+from datetime import timedelta
+from datetime import timezone
+
+import appdirs
+import click
+import logging_loki
+import mlflow
+import mrc
+import pandas as pd
+from dfp.stages.dfp_file_batcher_stage import DFPFileBatcherStage
+from dfp.stages.dfp_file_to_df import DFPFileToDataFrameStage
+from dfp.stages.dfp_inference_stage import DFPInferenceStage
+from dfp.stages.dfp_mlflow_model_writer import DFPMLFlowModelWriterStage
+from dfp.stages.dfp_postprocessing_stage import DFPPostprocessingStage
+from dfp.stages.dfp_preprocessing_stage import DFPPreprocessingStage
+from dfp.stages.dfp_rolling_window_stage import DFPRollingWindowStage
+from dfp.stages.dfp_split_users_stage import DFPSplitUsersStage
+from dfp.stages.dfp_training import DFPTraining
+from dfp.stages.multi_file_source import MultiFileSource
+from dfp.utils.regex_utils import iso_date_regex
+
+from morpheus.cli.utils import get_log_levels
+from morpheus.cli.utils import get_package_relative_file
+from morpheus.cli.utils import load_labels_file
+from morpheus.cli.utils import parse_log_level
+from morpheus.common import FileTypes
+from morpheus.common import FilterSource
+from morpheus.config import Config
+from morpheus.config import ConfigAutoEncoder
+from morpheus.config import CppConfig
+from morpheus.pipeline import LinearPipeline
+from morpheus.stages.general.monitor_stage import MonitorStage
+from morpheus.stages.output.write_to_file_stage import WriteToFileStage
+from morpheus.stages.postprocess.filter_detections_stage import FilterDetectionsStage
+from morpheus.stages.postprocess.serialize_stage import SerializeStage
+from morpheus.utils.column_info import ColumnInfo
+from morpheus.utils.column_info import DataFrameInputSchema
+from morpheus.utils.column_info import DateTimeColumn
+from morpheus.utils.column_info import DistinctIncrementColumn
+from morpheus.utils.column_info import IncrementColumn
+from morpheus.utils.column_info import RenameColumn
+from morpheus.utils.column_info import StringCatColumn
+from morpheus.utils.file_utils import date_extractor
+from morpheus.utils.logger import TqdmLoggingHandler
+from morpheus.utils.logger import set_log_level
+
+
+def configure_logging(log_level: int, loki_url: str):
```

Comment:
Instead of duplicating this function, lets add an argument to the `configure_logging` function which allows users to specify additional handlers. Then we can use that function here along with the loki handler and avoid duplicated code.